src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -46,4 +53,22 @@ public function run($request)
         }
         ImageThumbnailHelper::singleton()->run();
     }
+
+    /**
+     * TODO Refactor this whole mess into Symfony Console on a TaskRunner level,
+     * with a thin wrapper to show coloured console output via a browser:
+     * https://github.com/silverstripe/silverstripe-framework/issues/5542
+     * @throws \Exception
+     */
+    protected function addLogHandlers()
+    {
+        if ($logger = Injector::inst()->get(LoggerInterface::class)) {
```

Comment:
could we have a prototype specialisation, like
```
  Psr\Log\LoggerInterface.monolog:
    type: prototype
    class: Monolog\Logger
```
rather than taking the service and mutate its global state in here?